### PR TITLE
docs: change old documentation links of packages in README files

### LINKED ISSUE #11995: [firebase_core] Link for "Getting Started" section on README should be updated
Link for "Getting Started" section on README should be updated. 

When we open the link, it tells us to go [here](https://firebase.google.com/docs/flutter/setup?platform=ios)

### PR DESCRIPTION
**Pull request by ashwinair**
## Description

Changes old [firebase.flutter.dev/docs](https://firebase.flutter.dev/docs/) links to new [firebase.google.com/docs](https://firebase.google.com/docs)
of these packages:

- cloud_firestore
- cloud_firestore_web
- cloud_functions
- cloud_functions_web
- firebase_analytics
- fiebase_app_check_web
- firebase_app_installations_web
- firebase_app_installations
- firebase_core
- firebase_database
- firebase_database_web
- firebase_dynamic_links
- firebase_in_app_messaging
- firebase_messaging
- firebase_messaging_web
- firebase_ml_model_downloader
- firebase_performance
- firebase_remote_config_web
- firebase_storage
- firebase_storage_web

## Related Issues

closes #11995

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`).
This will ensure a smooth and quick review process. Updating the `pubspec.yaml` and changelogs is not required.

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [ ] My PR includes unit or integration tests for *all* changed/updated/fixed behaviors (See [Contributor Guide]).
- [ ] All existing and new tests are passing.
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [x] The analyzer (`melos run analyze`) does not report any problems on my PR.
- [x] I read and followed the [Flutter Style Guide].
- [x] I signed the [CLA].
- [x] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Does your PR require plugin users to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change.
- [x] No, this is *not* a breaking change.

[issue database]: https://github.com/flutter/flutter/issues
[Contributor Guide]: https://github.com/firebase/flutterfire/blob/master/CONTRIBUTING.md
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[pub versioning philosophy]: https://dart.dev/tools/pub/versioning
[CLA]: https://cla.developers.google.com/

### CODE REVIEW

File: packages/firebase_messaging/firebase_messaging_web/README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ The web implementation of `firebase_messaging`.
 
 ## Getting Started
 
-To get started with Cloud Messaging Web, please [see the documentation](https://firebase.flutter.dev/docs/messaging/overview)
+To get started with Cloud Messaging Web, please [see the documentation](https://firebase.google.com/docs/cloud-messaging)
 available at [https://firebase.flutter.dev](https://firebase.flutter.dev)
 
 Once installed, Cloud Messaging needs to be configured for Web Installation.  Please [see the documentation](https://firebase.flutter.dev/docs/messaging/overview#3-web-only-add-the-sdk) on Web Installation
```
Comment (russellwheatley) [inline packages/firebase_messaging/firebase_messaging_web/README.md:10]:
```suggestion

```

File: packages/firebase_database/firebase_database_web/README.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ The web implementation of `firebase_database`.
 
 ## Getting Started
 
-To get started with Firebase Realtime Database Web, please [see the documentation](https://firebase.flutter.dev/docs/database/overview)
+To get started with Firebase Realtime Database Web, please [see the documentation](https://firebase.google.com/docs/database/flutter/start)
 available at [https://firebase.flutter.dev](https://firebase.flutter.dev)
```
Comment (russellwheatley) [inline packages/firebase_database/firebase_database_web/README.md:8]:
```suggestion

```

File: packages/firebase_database/firebase_database_web/README.md [hunk 3]
```diff
@@ -4,7 +4,7 @@ The web implementation of `firebase_database`.
...
 
 Once installed, Firebase Realtime Database needs to be configured for Web Installation.  Please [see the documentation](https://firebase.flutter.dev/docs/database/overview#3-web-only-add-the-sdk) on Web Installation
```
Comment (russellwheatley) [inline packages/firebase_database/firebase_database_web/README.md:10]:
```suggestion

```

File: packages/firebase_app_installations/firebase_app_installations_web/README.md [hunk 4]
```diff
@@ -4,7 +4,7 @@ The web implementation of `firebase_app_installations`.
 
 ## Getting Started
 
-To get started with Firebase Installations Web, please [see the documentation](https://firebase.flutter.dev/docs/installations/overview)
+To get started with Firebase Installations Web, please [see the documentation](https://firebase.google.com/docs/projects/manage-installations#flutter)
 available at [https://firebase.flutter.dev](https://firebase.flutter.dev)
```
Comment (russellwheatley) [inline packages/firebase_app_installations/firebase_app_installations_web/README.md:8]:
```suggestion

```

File: packages/firebase_app_installations/firebase_app_installations_web/README.md [hunk 5]
```diff
@@ -4,7 +4,7 @@ The web implementation of `firebase_app_installations`.
...
 
 Once installed, Firebase Installations needs to be configured for Web Installation.  Please [see the documentation](https://firebase.flutter.dev/docs/installations/overview#3-web-only-add-the-sdk) on Web Installation
```
Comment (russellwheatley) [inline packages/firebase_app_installations/firebase_app_installations_web/README.md:10]:
```suggestion

```

File: packages/firebase_app_check/firebase_app_check_web/README.md [hunk 6]
```diff
@@ -5,7 +5,7 @@ The web implementation of `firebase_app_web`.
 ## Getting Started
 
 To get started with Firebase App Check on Web,
-please [see the documentation](https://firebase.flutter.dev/docs/app-check/overview)
+please [see the documentation](https://firebase.google.com/docs/app-check)
 available at [https://firebase.flutter.dev](https://firebase.flutter.dev)
```
Comment (russellwheatley) [inline packages/firebase_app_check/firebase_app_check_web/README.md:9]:
```suggestion

```

File: packages/firebase_app_check/firebase_app_check_web/README.md [hunk 7]
```diff
@@ -5,7 +5,7 @@ The web implementation of `firebase_app_web`.
...
 
 Once installed, Firebase  App Check needs to be configured for Web Installation.
```
Comment (russellwheatley) [inline packages/firebase_app_check/firebase_app_check_web/README.md:11]: You can also remove this

File: packages/firebase_analytics/firebase_analytics_web/README.md [hunk 8]
```diff
@@ -5,7 +5,7 @@ The web implementation of `firebase_analytics`.
 ## Getting Started
 
 To get started with Google Analytics for Firebase on Web,
-please [see the documentation](https://firebase.flutter.dev/docs/analytics/overview)
+please [see the documentation](https://firebase.google.com/docs/analytics/get-started?platform=flutter)
 available at [https://firebase.flutter.dev](https://firebase.flutter.dev)
```
Comment (russellwheatley) [inline packages/firebase_analytics/firebase_analytics_web/README.md:9]:
```suggestion

```

File: packages/firebase_analytics/firebase_analytics_web/README.md [hunk 9]
```diff
@@ -5,7 +5,7 @@ The web implementation of `firebase_analytics`.
...
 
 Once installed, Google Analytics for Firebase needs to be configured for Web Installation.
```
Comment (russellwheatley) [inline packages/firebase_analytics/firebase_analytics_web/README.md:11]: Please remove this.

File: packages/cloud_functions/cloud_functions_web/README.md [hunk 10]
```diff
@@ -5,7 +5,7 @@ The web implementation of `cloud_functions`.
 ## Getting Started
 
 To get started with Cloud Functions for Firebase on Web,
-please [see the documentation](https://firebase.flutter.dev/docs/functions/overview)
+please [see the documentation](https://firebase.google.com/docs/functions/get-started?gen=2nd)
 available at [https://firebase.flutter.dev](https://firebase.flutter.dev)
```
Comment (russellwheatley) [inline packages/cloud_functions/cloud_functions_web/README.md:9]:
```suggestion

```

File: packages/cloud_functions/cloud_functions_web/README.md [hunk 11]
```diff
@@ -5,7 +5,7 @@ The web implementation of `cloud_functions`.
...
 
 Once installed, Cloud Functions for Firebase needs to be configured for Web Installation.
```
Comment (russellwheatley) [inline packages/cloud_functions/cloud_functions_web/README.md:11]: Please remove this.

File: packages/cloud_firestore/cloud_firestore_web/README.md [hunk 12]
```diff
@@ -4,8 +4,8 @@ The web implementation of `cloud_firestore`.
 
 ## Getting Started
 
-To get started with Cloud Firestore Web, please [see the documentation](https://firebase.flutter.dev/docs/firestore/overview)
-available at [https://firebase.flutter.dev](https://firebase.flutter.dev)
+To get started with Cloud Firestore Web, please [see the documentation](https://firebase.google.com/docs/firestore/quickstart)
+available at [https://firebase.google.com](https://firebase.google.com/docs/flutter/setup?platform=ios#available-plugins)
 
 Once installed, Firestore needs to be configured for Web Installation.  Please [see the documentation](https://firebase.flutter.dev/docs/firestore/overview#3-web-only-add-the-sdk) on Web Installation
```
Comment (russellwheatley) [inline packages/cloud_firestore/cloud_firestore_web/README.md:11]:
```suggestion

```

**Review by russellwheatley — CHANGES_REQUESTED** [review]
Hey, thanks for the PR! I've made some more suggestions to remove links to deprecated website.

**Comment by ashwinair** [comment]
Hi @russellwheatley :)

I removed the lines as you suggested. can you have a look and tell me if there is anything else I have to edit.

**Comment by russellwheatley** [comment]
Sorry, @ashwinair, a PR has been merged that conflicts with this file `packages/cloud_firestore_odm/cloud_firestore_odm_generator/README.md`, do you mind resolving the conflict, please? Thanks 🙏

**Comment by ashwinair** [comment]
> do you mind resolving the conflict, please? Thanks 🙏

@russellwheatley, resolved.

**Review by russellwheatley — APPROVED** [review]

**Review by exaby73 — APPROVED** [review]

**Pull request merged**